### PR TITLE
libvpx: add support for msvc17 193

### PR DESCRIPTION
**Pull request by paulharris**
A simple patch to add support for MSVC17

A quick and simple follow-up to #13799

### CODE REVIEW

File: recipes/libvpx/all/conanfile.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def _execute_configure(self):
         elif is_msvc(self):
             vc_version = str(self.settings.compiler.version)
             if Version(self.settings.compiler.version) > "100": # New msvc compiler to triplet used in configure script for vpx
-                vc_version = {"190": "14", "191": "15", "192": "16"}[vc_version]
+                vc_version = {"190": "14", "191": "15", "192": "16", "193": "17"}[vc_version]
```
Comment (SpaceIm) [inline recipes/libvpx/all/conanfile.py:163]:
```suggestion
                vc_version = {"170": "11", "180": "12", "190": "14", "191": "15", "192": "16", "193": "17"}[vc_version]
```

Comment (SpaceIm) [reply recipes/libvpx/all/conanfile.py:163]: Could you also remove this ConanInvalidConfiguration you have added in last PR at the end of this branching? ConanInvalidConfiguration should only be called in validate() and there is already a logic in validate() checking that we cannot use other compilers not managed be current logic in `_execute_configure()`.

File: recipes/libvpx/all/conanfile.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ def _execute_configure(self):
         elif is_msvc(self):
             vc_version = str(self.settings.compiler.version)
             if Version(self.settings.compiler.version) > "100": # New msvc compiler to triplet used in configure script for vpx
```
Comment (SpaceIm) [inline recipes/libvpx/all/conanfile.py:162]: what does it mean?

File: recipes/libvpx/all/conanfile.py (hunk 1 above)
Comment (paulharris) [reply recipes/libvpx/all/conanfile.py]: Ok good point :)

File: recipes/libvpx/all/conanfile.py (hunk 2 above)
Comment (paulharris) [reply recipes/libvpx/all/conanfile.py:162]: True!

**Review by prince-chrismc — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**